src/App.js

```diff
@@ -2,15 +2,19 @@ import React, { useState } from "react";
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 
-const App = () => {
+const useSemiPersistentState = () => {
   const [todoList, setTodoList] = useState(
     JSON.parse(localStorage.getItem("savedTodoList")) || []
```

Comment:
Good idea to set it equal to an equal array, just in case the JSON doesn't parse correctly or there's nothing in local storage.